homeassistant/components/netgear/__init__.py

```diff
@@ -203,3 +203,24 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 async def update_listener(hass: HomeAssistant, config_entry: ConfigEntry) -> None:
     """Handle options update."""
     await hass.config_entries.async_reload(config_entry.entry_id)
+
+
+async def async_remove_config_entry_device(
+    hass: HomeAssistant, config_entry: ConfigEntry, device_entry: dr.DeviceEntry
+) -> bool:
+    """Remove a config entry from a device."""
+    router = hass.data[DOMAIN][config_entry.entry_id][KEY_ROUTER]
+
+    device_mac = None
+    for connection in device_entry.connections:
+        if connection[0] == "mac":
```

Comment:
I suggest using the `CONNECTION_NETWORK_MAC` constant from the device registry helper.

Reply:
Adressed in PR: https://github.com/home-assistant/core/pull/87991